tests/flytekit/unit/core/image_spec/test_default_builder.py

```diff
@@ -28,6 +28,7 @@ def test_create_docker_context(tmp_path):
         requirements=os.fspath(other_requirements_path),
         source_root=os.fspath(source_root),
         commands=["mkdir my_dir"],
+        entrypoint=["/bin/bash"],
```

Comment:
Can you add another test case to check the behavior of `entrypoint=[]`? 

(You can use `test_create_docker_context_with_git_subfolder` for a quick way to setup `create_docker_context`.)

Reply:
Ohhhh because empty lists are falsey, good catch! I found out some other niche behavior in Docker but this should be better now..